### PR TITLE
fix(copy): update guid reminder and trading limits copy and styles

### PR DESCRIPTION
**Pull request by schnogz**
## Description 
- Updates copy on wallet id reminders
- Updates and styles copy on Trading Limits flyout

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 1]
```diff
@@ -1505,7 +1506,7 @@ type MessagesType = {
   'modals.tradinglimits.buy_with_a_card': 'Buy with a Card'
   'modals.tradinglimits.deposits_and_withdrawals': 'Deposits & Withdrawals'
   'modals.tradinglimits.earn_interest': 'Earn Interest'
-  'modals.tradinglimits.unlock_gold_tier': 'Unock Gold Tier ->'
+  'modals.tradinglimits.unlock_gold_tier': 'Unlock Gold Tier ->'
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:1509]: 😱

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
+  Text,
+  TextGroup
+} from 'blockchain-info-components'
+import { fiatToString } from 'core/exchange/currency'
+import { FlyoutWrapper } from 'components/Flyout'
+import { UserTierType } from 'data/types'
+import { WalletFiatType } from 'core/types'
+
+import { ITEMS, TIER_TYPES, TIERS } from './model'
+import { Props as OwnProps, SuccessStateType } from '.'
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:25]: 👏

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx [hunk 3]
```diff
@@ -64,6 +70,7 @@ const FooterWrapper = styled(FlyoutWrapper)`
   justify-content: flex-end;
   display: flex;
   flex-direction: column;
+  margin-top: -16px;
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:73]: Nothing wrong with this, but one thing I've been doing lately is using `transform: translateY(-16px)` as it won't affect the elements around the item you're moving.

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx [hunk 4]
```diff
@@ -266,12 +279,12 @@ const Template: React.FC<Props> = props => {
                   })
                 }}
               />
-            </Text>
+            </ItemSubtitle>
 
-            <Text color='grey600' size='12px' weight={500}>
+            <Text color='grey600' lineHeight='1.5' size='12px' weight={500}>
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:284]: need the `%` symbol or no?

**Review by blockdylanb — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts (hunk 1 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:1509]: 👏

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx (hunk 4 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:284]: in this case no need, if you are about to say 150 then you need to set like `150%`

**Pull request merged**